config.template.js

```diff
@@ -24,5 +24,6 @@ window.env = {
   RESOURCE_API_HOST: '$RESOURCE_API_HOST',
   SEARCH_SERVICE_HOST: '$SEARCH_SERVICE_HOST',
   USE_DEMO_LOGO: '$USE_DEMO_LOGO',
-  CATALOG_PORTAL_BASE_URI: '$CATALOG_PORTAL_BASE_URI'
+  CATALOG_PORTAL_BASE_URI: '$CATALOG_PORTAL_BASE_URI',
+  ACCESS_REQUEST_API_HOST: '$ACCESS_REQUEST_API_HOST'
```

Comment:
Husk å legge inn env var I deployments (deploy mappa for hver miljø env.yaml)

Reply:
Bra spotta! [Add ACCESS_REQUEST_API_HOST to deploy env](https://github.com/Informasjonsforvaltning/fdk-portal/pull/2155/commits/7dc9e06b01d73be405ba53f6e926df7e45f6680d)